### ISSUE TITLE
the mAP

### ISSUE THREAD

**Issue by NooneOnlyOne** [buxiangzhiren/ContextLoc#6]
Hi ，I get the mAP (%) of tIoU thresholds from 0.1 to 0.9
the Flow result
![d0a2ab17b144604d94faa0857e7bfc1](https://user-images.githubusercontent.com/80327317/136153098-702ea6ba-eb21-49c6-85db-ca131579b0e2.jpg)
the RGB result
![8fea3ae3b3a3db9a4584e078cee835b](https://user-images.githubusercontent.com/80327317/136153382-d6c2e082-5f99-4818-b598-c8ba065b463b.png)
and `run test_two_stream.sh ` the result 
![af79af73d131d6b74e2f68f1b96c456](https://user-images.githubusercontent.com/80327317/136153986-0113166a-005d-4cd4-a5d5-1105a8c849ce.jpg)

Why is there a big gap between the results I got and the results of your paper, especially Flow result and the final result

**Comment by buxiangzhiren** [comment]
I think it is the problem of the epoch. You can try the checkpoint of the 45-th epoch.

**Comment by NooneOnlyOne** [comment]
> I think it is the problem of the epoch. You can try the checkpoint of the 45-th epoch.

I used the checkpoint of the 45-th epoch of Flow and RGB  and During flow and RGB training and testing, the ft_path in the PGCNDataSet class was changed. So I get the above resluts

**Comment by buxiangzhiren** [comment]
Yeah, your path is right. You can try the best checkpoint or other checkpoints. By the way,
Are your parameter settings correct? Such as the pytorch version and the learning rate. If your answer is yes, you can run the code again. The results of others who run the code don’t have a such big performance gap like yours.

**Comment by NooneOnlyOne** [comment]
Thsnks，I think I have found the problem. I changed the batch_size to 128, so there is a big gap with your paper. However, I still have a question, why use checkpoint of the 45-th epoch of Flow to ` run test.sh` multiple times and get different results
![1633678677(1)](https://user-images.githubusercontent.com/80327317/136517647-33683665-4a01-4a96-8cf5-07f3cf55e156.jpg)

**Comment by buxiangzhiren** [comment]
I also found such a problem which may come from the proposal sampling of PGCN. And I think it is in normal variance.

**Comment by NooneOnlyOne** [comment]
> I also found such a problem which may come from the proposal sampling of PGCN. And I think it is in normal variance.

I think your work is well done. I am very interested in your method. Can you give me the code for the Anet1.3 dataset? I want to do some work on your basis. If you can, my email is`15163881521@163.com`, Thanks

**Comment by buxiangzhiren** [comment]
Thank you for your attention to our work. Since some files in Anet1.3 involve my follow-up work, I cannot give you. But you can simply reproduce the code based on pgcn and a suitable video-level classification network.

**Comment by buxiangzhiren** [comment]
In addition, you also can get a stronger baseline by adding l-net and g-net into any open source two-stage method.

**Comment by NooneOnlyOne** [comment]
> Thank you for your attention to our work. Since some files in Anet1.3 involve my follow-up work, I cannot give you. But you can simply reproduce the code based on pgcn and a suitable video-level classification network.

ok，thanks
